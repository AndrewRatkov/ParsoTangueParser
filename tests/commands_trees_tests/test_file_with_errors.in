if then
id := 30;
fi