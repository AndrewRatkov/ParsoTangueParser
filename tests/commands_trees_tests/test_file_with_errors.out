IF
|
+--->[ERR,Empty expression]
|
THEN
|
+--->[ERR Variable should have been initialized before]
|
FI
